compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt

```diff
@@ -66,6 +66,18 @@ class ComposeUIViewControllerConfiguration {
      * explanation on how to fix the issue.
      */
     var enforceStrictPlistSanityCheck: Boolean = true
+
+    /**
+     * If set to true, the Compose will encode the rendering commands on a dedicated render thread,
+     * when possible.
+     * This can improve the performance when no interop UIKit is used.
+     *
+     * It's an experimental API, and the effects of enabling it are not considered stable.
+     *
+     * Changing this setting outside of `ComposeUIViewController` `configure` argument scope has no effect.
+     */
+    @ExperimentalComposeApi
```

Comment:
`ExperimentalComposeApi` is from `runtime` module. We should use `ExperimentalComposeUiApi` instead.

Reply:
Not in this PR, but we need to revisit other experimental annotations for iOS APIs (there are a few in this file at least)
Probably it's for @ASalavei?

Reply:
I think we discussed it on `Stabilization` meeting. It indeed will go away. This newly introduced is indeed experimental and will be removed either way later (will become the only strategy, seemingly there is no disadvantages to it being always on)